pmd-cataloger/src/main/java/sfdc/sfdx/scanner/pmd/XmlFileFinder.java

```diff
@@ -34,39 +35,66 @@
 		}
 	}
 
+	/**
+	 * Represents a container for XML. This can either be a Jar file which may contain multiple XML files or an XML
+	 * file that contains itself.
+	 */
+	public static final class XmlContainer {
+		public final String filePath;
+		public final List<String> containedFilePaths;
```

Comment:
I'd love to see commenting for what these variables mean. Like `filePath is the path to the file, and if the file is a JAR then containedFilePaths will be all the XMLs inside that JAR`, or something like that.